impedance/models/circuits/fitting.py

```diff
@@ -144,28 +180,29 @@ class BasinhoppingBounds(object):
             https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.basinhopping.html
             """
 
-            def __init__(self, xmax=upper_bounds, xmin=lower_bounds):
+            def __init__(self, xmax=bounds[1], xmin=bounds[0]):
                 self.xmax = np.array(xmax)
                 self.xmin = np.array(xmin)
 
             def __call__(self, **kwargs):
-                x = kwargs["x_new"]
+                x = kwargs['x_new']
                 tmax = bool(np.all(x <= self.xmax))
                 tmin = bool(np.all(x >= self.xmin))
                 return tmax and tmin
 
         basinhopping_bounds = BasinhoppingBounds()
```

Comment:
Could this be clearer? `basinhopping_bounds = BasinhoppiingBounds(xmax=bounds[1], xmin=bounds[0])`

And then line 183 wouldn't need defaults and becomes just `def __init__(self, xmax, xmin):`

Reply:
Good call! Changed.